mvlearn/embed/kcca.py

```diff
@@ -175,6 +202,8 @@ def __init__(
         reg=0.1,
         decomp='full',
         method='kettenring-like',
```

Comment:
can you reorder the params either here or in the docstrings so they're in the same order?

Reply:
arent they already in the same order?

Reply:
The init has the order "n_components, ktype, constant, sigma, degree, reg, decomp, method, mrank, precision" but it looks like the docstrings have "n_components, ktype, degree, constant, sigma, reg, decomp, method, mrank, precision". So the params 3,4,5 have different order

Reply:
oh my bad. missed that